sdk/src/electrs.ts

```diff
@@ -1,7 +1,22 @@
+/**
+ * Base path for the mainnet Esplora API.
+ * @default "https://btc-mainnet.interlay.io"
+ */
 export const MAINNET_ESPLORA_BASE_PATH = "https://btc-mainnet.interlay.io";
```

Comment:
@gregdhill @nakul1010 can you coordinate with @ns212 to move these to a gobob.xyz domain.

Reply:
Sure, keeping it for now until we add it on explorer.

Reply:
Issue created: #51